src/typescript/AzureFunctions.Blob/02UsingPlainBlobOutputBindings/.funcignore

```diff
@@ -0,0 +1,7 @@
+*.js.map
+*.ts
+.git*
+.vscode
+local.settings.json
```

Comment:
I would be OK with including local.settings.json in source control as long as these files do not contain secrets. I see that you named the files `sample.local.settings.json` but I didn't see any instructions in the lesson to rename them. Ideally I'd prefer the project to build without any instructions for the user, so as long as the settings don't contain anything secret you can include them.

Reply:
Done